package.json

```diff
@@ -17,14 +18,18 @@
   ],
   "license": "MIT",
   "bin": {
-    "gramps": "bin/index.js"
+    "gramps": "gramps.js"
   },
   "scripts": {
-    "gramps": "bin/index.js",
+    "gramps": "./gramps.js",
     "prepush": "npm test",
-    "lint": "eslint {bin, gateway, lib, test}/**/*.js",
+    "build": "babel src -d .",
```

Comment:
Should this be `babel src -d dist` to put the built artifact in a separate directory? It's a bit confusing to have a `gateway/`, `lib/`, `gramps.js` added to the root of the repo after build.

Reply:
I put them in the root to make it easier for anyone looking at their `node_modules`. I don't have a particularly strong preference, but I always like it when I open up a package and have the main files at `/node_modules/packagename/index.js`.

Reply:
https://github.com/apollographql/graphql-tools/blob/master/package.json#L5
https://github.com/facebook/dataloader/blob/master/package.json#L68
https://github.com/petkaantonov/bluebird/blob/master/package.json#L69

It looks like most packages have a dist/release directory. I looked at these three and also `recompose` which I couldn't immediately figure out to be honest.

interestingly it looks like `dataloader` has an `index.js` at it's root that just calls into `dist/`

Reply:
Okay, cool. I moved them to `dist` and updated the `.gitignore` to match.